src/components/topList/TopList.jsx

```diff
@@ -1,13 +1,60 @@
-import React from 'react';
-import Song from '../song/Song';
+import React from 'react'
+import PropTypes from 'prop-types'
+import Song from '../song/Song'
+import './_TopList.scss'
 
-const TopList = () => {
-  return <div>
-    <h2>Top J-Pop</h2>
-    <Song />
-    <Song />
-    <Song />
+const TopList = ({ genre }) => {
+  return <div className="top-list">
+    <div className="top-list__header">
+      <h2>Top {genre}</h2>
```

Comment:
What if in the future you want to change the copy? Probably it’s better to pass the whole title as a prop 😅

Reply:
Fixed